src/components/Article-votes.jsx

```diff
@@ -0,0 +1,92 @@
+import { useContext, useEffect, useState } from "react"
+import LoggedInContext from "../contexts/Logged-In-User-Context"
+import { patchArticleById } from "../utils/PATCH-article-votes"
+
+const ArticleVotes = (props) => {
+   
+    const articleVotes = props.articleVotes
+    const setArticleVotes = props.setArticleVotes
+    const currentArticle = props.currentArticle
+    
+    const { loggedInUser, setLoggedInUser } = useContext(LoggedInContext)
+    const [hasVoted, setHasVoted] = useState(false)
+    const [increment, setIncrement] = useState(0)
+    const [isError, setIsError] = useState(false)
+
+    useEffect(() => {
+        setArticleVotes(currentArticle.votes)
```

Comment:
why don't we set the initial state of this to be the current article votes when its declared?